python/Ganga/Runtime/GPIexport.py

```diff
@@ -18,6 +18,31 @@
 from Ganga.GPIDev.Base.Objects import GangaObject
 from inspect import isclass
 
+def _addToInterface(interface, name, _object):
+
+    if isType(_object, GangaObject):
+        setattr(interface, name, addProxy(_object))
+    elif isclass(_object) and issubclass(_object, GangaObject):
+        setattr(interface, name, getProxyClass(_object))
+    else:
+        setattr(interface, name, _object)
+
+def exportToInterface(myInterface, name, _object, doc_section, docstring=None):
```

Comment:
You can get `jobs` with:
```python
from Ganga.Core.GangaRepository import getRegistry
jobs = getRegistry('jobs')
```
and for the rest of them we *can* know. Perhaps this is something which we will have to tackle on a case-by-case basis but it's certainly possible (even if it requires changes to the bootstrap like `queues` did) and it's also correct.